server/athenian/api/controllers/features/entries.py

```diff
@@ -288,273 +551,46 @@ def _compose_cache_key_participants(participants: List[PRParticipants]) -> str:
-        _compose_cache_key_repositories(devs),  # yes, _repositories
-        labels, jira,
-        release_settings,
-    ),
-)
-async def calc_developer_metrics_github(devs: Sequence[Collection[str]],
-                                        repositories: Sequence[Collection[str]],
-                                        time_intervals: Sequence[Sequence[datetime]],
-                                        topics: Set[DeveloperTopic],
-                                        labels: LabelFilter,
-                                        jira: JIRAFilter,
-                                        release_settings: ReleaseSettings,
-                                        account: int,
-                                        meta_ids: Tuple[int, ...],
-                                        mdb: Database,
-                                        pdb: Database,
-                                        rdb: Database,
-                                        cache: Optional[aiomcache.Client],
-                                        ) -> Tuple[np.ndarray, List[DeveloperTopic]]:
-    """
-    Calculate the developer metrics on GitHub.
-
-    :return: repositories x granularities x devs x time intervals x topics.
-    """
-    all_devs = set(chain.from_iterable(devs))
-    all_repos = set(chain.from_iterable(repositories))
-    if not all_devs or not all_repos:
-        return np.array([]), list(topics)
-    time_from, time_to = time_intervals[0][0], time_intervals[0][-1]
-    mined_dfs = await mine_developer_activities(
-        all_devs, all_repos, time_from, time_to, topics, labels, jira, release_settings,
-        account, meta_ids, mdb, pdb, rdb, cache)
-    topics_seq = []
-    arrays = []
-    repo_grouper = partial(group_by_repo, developer_repository_column, repositories)
-    developer_grouper = partial(group_actions_by_developers, devs)
-    for mined_topics, mined_df in mined_dfs:
-        topics_seq.extend(mined_topics)
-        calc = DeveloperBinnedMetricCalculator([t.value for t in mined_topics], (0, 1))
-        groups = group_to_indexes(mined_df, repo_grouper, developer_grouper)
-        arrays.append(calc(mined_df, time_intervals, groups))
-    result = np.full(arrays[0].shape, None)
-    result.ravel()[:] = [
-        [list(chain.from_iterable(m)) for m in zip(*lists)]
-        for lists in zip(*(arr.ravel() for arr in arrays))
-    ]
-    result = result.swapaxes(1, 2)
-    return result, topics_seq
-
-
-METRIC_ENTRIES = {
-    "github": {
-        "prs_linear": calc_pull_request_metrics_line_github,
-        "prs_histogram": calc_pull_request_histograms_github,
-        "code": calc_code_metrics_github,
-        "developers": calc_developer_metrics_github,
-        "releases_linear": calc_release_metrics_line_github,
-    },
-}
+        mod = importlib.import_module(f"{base_module}.{variation}")
+    except ModuleNotFoundError:
+        if raise_err:
+            raise
+
+        log.warning(
```

Comment:
This is clearly an error.
```suggestion
        log.error(
```